toolkit/imageconfigs/scripts/setup_cvm_image.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+EFIDIR="BOOT"
+KERNEL_VERSION=""
+
+get_kernel_version() {
```

Comment:
looks a bit hacky. can we think of a more stable way. looks like we want this for "vmlinuz-uki.efi".

Does it make sense to either query using a command, or even add the version to "vmlinuz-uki.efi" when we build the respective package?

Reply:
Ex. in UKI SPEC build and file section
%files
/lib/modules/%{kernelver}/vmlinuz-uki.efi

=> %{kernelver} as file name?

+ In this script, can we query the location/version of this binary by checking the version of the UKI package?